bower.json

```diff
@@ -22,7 +22,9 @@
     "angular-oauth2": "3.0.1",
     "angular-filter": "~0.5.4",
     "angular-dragdrop": "1.0.9",
-    "angular-ui-sortable": "~0.13.4"
+    "angular-ui-sortable": "~0.13.4",
+    "ng-file-upload": "~5.0.9",
+    "mdi": "~1.1.70"
```

Comment:
we already have imported the font icons, in case you need the svgs then use material-design-icons